core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/content/components/dot-edit-page-toolbar/dot-edit-page-toolbar.component.html

```diff
@@ -5,6 +5,11 @@
             [url]="pageState.page.pageURI"
             [apiLink]="apiLink"
         ></dot-edit-page-info>
+        <!-- <dot-icon-button
```

Comment:
remove ?

Reply:
this is hidden while it gets ready for release